devito/arch/compiler.py

```diff
@@ -470,9 +470,11 @@ def __init_finalize__(self, **kwargs):
         GNUCompiler.__init_finalize__(self, **kwargs)
         platform = kwargs.pop('platform', configuration['platform'])
 
-        # Graviton flag
```

Comment:
This was redundant, duplicating `mcpu` flag

Reply:
would probably remove the GNU one instead then. Don't remember if can end up in the GNUCompiler but not in the Arm one for graviton

Reply:
This way you are safe for both DEVITO_ARCH={gcc, arm} and PLATFORM=graviton

Reply:
right, checked with all possible env combos for compiler{GCC, arm}, and platform={graviton, cortexa76} seems to that all flags are ok